python/pyspark/sql/column.py

```diff
@@ -251,15 +285,16 @@ def __iter__(self):
 
     # string methods
     _rlike_doc = """
-    Return a Boolean :class:`Column` based on a regex match.
+    SQL RLIKE expression (LIKE with Regex). Returns a boolean :class:`Column` based on a regex
```

Comment:
Problem is that in Scala or Java users get regular expressions dialect they expect. In Python they don't. 

But fair enough. Let's leave it for another time.